### ISSUE TITLE
Ktlint formatting is incosistent with what is reporting

### ISSUE THREAD

**Issue by PPetka** [pinterest/ktlint#877]
This one feels critical to me and, and I guess should be hotfixed
Fixing [this one ](https://github.com/pinterest/ktlint/issues/796) might introduce this issue.

I am having file that according to AS is formatted OK.
```
object Someclass : List<Shortcut> by listOf(
    Shortcut(
        id = "1",
        url = "url"
    ),
    Shortcut(
        id = "2",
        url = "asd"
    ),
    Shortcut(
        id = "3",
        name = "TV"
    )
)
```
Ktlint found error:
Unexpected indentation (expected 0, actual 4)
And formatted this like:

```
object SomeClass : List<Shortcut> by listOf(
        Shortcut(
            id = "1",
            name = "Sport"
        ),
        Shortcut(
            id = "2",
            name = "Nöje"
        ),
        Shortcut(
            id = "3",
            name = "TV"
        )
    )
```
After I run ktlint again I am getting error 
Unexpected indentation (expected 4, actual 8)

So even ktlint made formatting, it still reports error.
Its now arguing with itself

version 0.38.1

**Comment by romtsn** [comment]
It's actually parameter-list-wrapping, you can try to disable it for this block for now

**Comment by cowst** [comment]
> It's actually parameter-list-wrapping, you can try to disable it for this block for now

Nope, I have the same issue and only disabling the (now experimental) argument-list-wrapping I get no errors.
So it has been tagged correctly :)
